pybandits/quantitative_model/zooming.py

```
@@ -266,25 +269,18 @@ def _validate_segments(self):
 
     def model_post_init(self, __context: Any) -> None:
         self._validate_segments()
-        self._init_base_model()
         segment_models_types = set(type(model) if model is not None else None for model in self.sub_actions.values())
         if None in segment_models_types:
             if len(segment_models_types) > 1:
                 raise ValueError("All segments must either have a model or miss a model.")
             self.sub_actions = dict(
-                zip(self.sub_actions, [self._base_model.model_copy(deep=True) for _ in range(len(self.sub_actions))])
+                zip(self.sub_actions, [self.base_model.model_copy(deep=True) for _ in range(len(self.sub_actions))])
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)